src/gwt/src/org/rstudio/studio/client/workbench/ui/PaneManager.java

```diff
@@ -946,7 +1106,26 @@ private void restoreSavedLayout()
          window.onWindowStateChange(new WindowStateChangeEvent(WindowState.NORMAL, true));
 
       maximizedWindow_.onWindowStateChange(new WindowStateChangeEvent(WindowState.NORMAL, true));
-      resizeHorizontally(panel_.getWidgetSize(right_), widgetSizePriorToZoom_);
+      
+      ArrayList<Double> leftStart = new ArrayList<>();
+      ArrayList<Double> leftEnd = new ArrayList<>();
+      if (leftList_.size() > 0)
+      {
+         for (int i = 0; i < leftList_.size(); i++)
+            leftStart.add(0.0);
+
+         if (leftWidgetSizePriorToZoom_.size() != leftList_.size())
+            leftEnd = getValidColumnWidths(leftList_, leftWidgetSizePriorToZoom_, false);
+         else
+         {
+            // Copy leftWidgetSizePriorToZoom_ rather than passing it directly because it may be
+            // cleared before animation completes
+            for (Double size : leftWidgetSizePriorToZoom_)
+               leftEnd.add(size);
```

Comment:
Nit: `addAll` will do this in one line

Reply:
Doesn't `addAll` only do a shallow copy though?

Reply:
IntelliJ also recommends replacing with `addAll`.

Reply:
It does do a shallow copy, but this loop also does a shallow copy.

Reply:
You're right, but I think it's ok because this gets passed as a final parameter. I've updated to follow your nit.